azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/screens/ChatScreen.kt

```diff
@@ -112,6 +123,9 @@ internal fun ChatScreen(
                         )
                     }
                 }
+                if (viewModel.messageContextMenu != null) {
+                    showMessageContextMenu(viewModel.messageContextMenu, dispatch = viewModel.postAction)
```

Comment:
Show all the time, but in a closed state, call "createMessageContextMenu" instead of "show"